change/@itwin-map-layers-543b13e0-c523-48d6-85de-a8c9a0e77658.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix issue where two instances of 'Areal Imagery with labels' are visible in map layers widget",
```

```suggestion
  "comment": "Fix issue where two instances of 'Aerial Imagery with labels' are visible in map layers widget",
```
Nit: typo (I think?)